src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj

```diff
@@ -126,6 +129,7 @@
     <PackageReference Include="Microsoft.ServiceHub.Client" Version="$(MicrosoftServiceHubClientVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Debugger.Engine-implementation" Version="$(MicrosoftVisualStudioDebuggerEngineimplementationVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Designer.Interfaces" Version="$(MicrosoftVisualStudioDesignerInterfacesVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.Extensibility.Testing.SupportFiles" GeneratePathProperty="true" Version="$(MicrosoftVisualStudioExtensibilityTestingSupportFilesVersion)" PrivateAssets="all" />
```

Comment:
No. But it's also not worse than using reflection to pretend we're clean 🤣 

This _would_ have prevented the recent regression because the type forwarders have been present, and would have directed the compiler to bind against the proper implementation location long before the forwarders were removed.

Reply:
ok, but if this is just a different hack shouldn't we instead wait for the right fix of their being a real API? Reflection is concerning enough that we can track it. This looks like we are just paving over the issue until another problem arises.

Reply:
@jmarolf No, this approach is better because we can track the use of the reference assembly, and when we remove the reference assembly it will immediately identify all sections of code that need to be updated with the new API. The previous approach required manual value tracking of objects of type `string` and `object`.